libdnf/rpm/package.cpp

```diff
@@ -32,7 +32,7 @@ inline static std::string cstring2string(const char * input) {
 
 namespace libdnf::rpm {
 
-const char * Package::get_name_cstring() const noexcept {
+const char * Package::get_name_cstring() const {
```

Comment:
To add `const` is correct, but why did you removed `noexcept`? I suggest that there is no allocation or anything that could raise an error.

Reply:
You're looking at it from a wrong direction. I added a comment why noexcept MUST NOT be used in Package.

Reply:
Good point.